charts/seed-monitoring/charts/prometheus/templates/blackbox-exporter-config.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
 data:
   blackbox.yaml: |
     modules:
+      icmp_probe:
+        prober: icmp
+        timeout: 2s
```

Comment:
Maybe increase timeout to 5s to prevent false positives due to latency

Reply:
done